### PR TITLE
Remove the Preview banner

### PR DESCRIPTION
**Pull request by ezr-ondrej**
This should follow up enabling Content and Launch(#1313) in stable.

### CODE REVIEW

File: src/Components/LandingPage/LandingPage.tsx [hunk 1]
```diff
@@ -69,102 +64,71 @@ export const LandingPage = () => {
...
-              className="pf-u-font-weight-bold"
-            >
-              Build an Image with Custom Content
-            </Button>
-          </p>
-        </ExpandableSection>
-      )}
+      <ExpandableSection
+        className="pf-m-light pf-u-mb-xl expand-section"
+        toggleText="Help get started with Preview features"
```
Comment (lucasgarfield) [inline src/Components/LandingPage/LandingPage.tsx:69]:
```suggestion
        toggleText="Help get started with new features"
```

File: src/Components/LandingPage/LandingPage.tsx [hunk 2]
```diff
@@ -69,102 +64,71 @@ export const LandingPage = () => {
...
+        onToggle={setShowHint}
+        isExpanded={showHint}
+        displaySize="large"
+      >
+        <p className="pf-u-pb-sm">
+          For help getting started, access the quick starts for our New
```
Comment (lucasgarfield) [inline src/Components/LandingPage/LandingPage.tsx:75]:
```suggestion
          For help getting started, access the quick starts for our new
```

**Comment by lucasgarfield** [comment]
Looks good apart from two minor nitpicks!

**Comment by lucasgarfield** [comment]
If you have time, it might be nice to extract this into a `<Quickstarts>` component that could live in `Quickstarts.tsx`. No big deal if you want to just keep it inline for now though, up to you. :smile:

**Comment by ezr-ondrej** [comment]
> If you have time, it might be nice to extract this into a `<Quickstarts>` component that could live in `Quickstarts.tsx`

I'm happy to do it as a follow-up, but I'd love to keep the changes here minimal, I'm happy to work on that tho, sounds like a good idea ;)

**Review by lucasgarfield — APPROVED** [review]
Thank you for this!

**Pull request merged**